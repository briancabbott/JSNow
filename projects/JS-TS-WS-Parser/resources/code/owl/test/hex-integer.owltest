1, 1.2, 0x1, 0xF, 0xff, 0xff.f
---
y = x{','}
x = integer : int number : num
