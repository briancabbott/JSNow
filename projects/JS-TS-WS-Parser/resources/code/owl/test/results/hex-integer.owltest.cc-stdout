y (0 - 30)
  x : INT (0 - 1)
    integer - 1 (0 - 1)
  x : NUM (3 - 6)
    number - 1.200000 (3 - 6)
  x : INT (8 - 11)
    integer - 1 (8 - 11)
  x : INT (13 - 16)
    integer - 15 (13 - 16)
  x : INT (18 - 22)
    integer - 255 (18 - 22)
  x : NUM (24 - 30)
    number - 255.937500 (24 - 30)
