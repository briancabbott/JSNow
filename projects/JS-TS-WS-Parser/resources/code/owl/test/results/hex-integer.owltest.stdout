. 1    , 1.2  , 0x1  , 0xF  , 0xff , 0xff.f
  x:int  x:num  x:int  x:int  x:int  x:num-
